### ISSUE TITLE
support scanning partitioned tables

### ISSUE THREAD

**Issue by wuputah** [duckdb/pg_duckdb#19]
When obtaining metadata for a partitioned table, several tables are returned - the parent and then its children. Support scanning each table in turn (or in parallel).

For now, we can assume that each partition is a heap table, but in the future, each table might have a different storage method.

**Comment by ritwizsinha** [comment]
@JelteF  I would like to start working to support this feature, if that is okay

**Comment by JelteF** [comment]
I think this should already be supported, now that #477 is merged. If that's indeed the case all that's really needed is some tests.

**Comment by JelteF** [comment]
Yeah #477 included tests for partitioned tables already: https://github.com/duckdb/pg_duckdb/blob/de3735a963704a9bebd6cacd52f03e2ede572219/test/regression/expected/scan_postgres_tables.out#L162-L253

So closing this.
